### PR TITLE
Minor visual fixes

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it fixes various visual things in project, like search bar and product list.

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).

### CODE REVIEW

File: src/components/Overlay/index.tsx [hunk 1]
```diff
@@ -58,12 +59,20 @@ class OverlayProvider extends React.Component<
...
   children,
+  className,
   context: { type, theme, hide }
 }) => (
   <div
-    className={`overlay${type ? ` overlay--${type}` : ""}`}
+    className={classNames(
+      `overlay${type ? ` overlay--${type}` : ""}`,
+      className
+    )}
```
Comment (patrys) [inline src/components/Overlay/index.tsx]: Maybe:

```typescript
    className={classNames({
        overlay,
        [`overlay--${type}`]: type
        className
    })}
```

**Comment by patrys** [comment]
It would be nice if you included a screenshot for UI changes.

File: src/components/Overlay/index.tsx (hunk 1 above)
Comment (dominik-zeglen) [reply src/components/Overlay/index.tsx]: Uh, I was lazy and copied code written before 😞

**Comment by dominik-zeglen** [comment]
Product list displays now two columns on mobile devices. Also tile ratio was improved.
zrzut ekranu 2018-11-08 o 12 08 14

Social media icons are now displayed properly.
zrzut ekranu 2018-11-08 o 12 08 41

Search is now displayed as input if no query is given.
zrzut ekranu 2018-11-08 o 12 25 01

Got rid of overlay while searching products.
zrzut ekranu 2018-11-08 o 12 25 19

File: src/components/Overlay/index.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import classNames from "classnames";
```
Comment (weronka) [inline src/components/Overlay/index.tsx]: let's try to stick to alphabetical ordering imports in the project

File: src/components/SearchOverlay/scss/index.scss [hunk 3]
```diff
@@ -36,9 +41,21 @@
   }
 
   &__products {
-    max-height: calc(100vh - 9rem);
+    background: #ffffff;
```
Comment (weronka) [inline src/components/SearchOverlay/scss/index.scss]: use `$white` variable here

**Pull request merged**